pkg/auth/providers/common/ldap/ldap_util.go

```diff
@@ -170,7 +170,7 @@ func GetGroupSearchAttributesForLDAP(ObjectClass string, config *v3.LdapConfig)
 	return groupSeachAttributes
 }
 
-func AuthenticateServiceAccountUser(serviceAccountPassword string, serviceAccountUsername string, defaultLoginDomain string, lConn *ldapv3.Conn) error {
+func AuthenticateServiceAccountUser(serviceAccountPassword string, serviceAccountUsername string, defaultLoginDomain string, lConn ldapv3.Client) error {
```

Comment:
Is this getting called anywhere in the tests?  Not seeing it....if it's not being called, do we need to change the param?